public/global.css

```diff
@@ -40,28 +41,38 @@ input:focus {
 	outline: none;
 }
 
-
-@media screen and (min-width: 450px){
+@media screen and (min-width: 576px) {
+/* @media screen and (min-width: 450px){ */
```

Comment:
I don't think there's any need to keep the old media queries

Reply:
If we agree to keep the new ones, will remove old. Should I?

Reply:
I'd say so. Worst case, we can go back in git history, if we _really_ need the old ones back.

Reply:
Removed.